src/status_im/contexts/wallet/common/transaction_settings/gas_price/view.cljs

```diff
@@ -0,0 +1,47 @@
+(ns status-im.contexts.wallet.common.transaction-settings.gas-price.view
+  (:require [status-im.contexts.wallet.common.transaction-settings.view :as transaction-settings]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn hint-and-status
+  [suggested-gas-price current-gas-price {:keys [low high]} entered-value]
+  (cond
+    (> entered-value high)                {:hint-text (i18n/label :t/gas-price-higher
+                                                                  {:high high})
+                                           :status    :error}
+    (> entered-value suggested-gas-price) {:hint-text (i18n/label :t/gas-price-higher-than
+                                                                  {:current
+                                                                   suggested-gas-price})
+                                           :status    :warning}
+    (< entered-value low)                 {:hint-text (i18n/label :t/gas-price-lower
+                                                                  {:low low})
+                                           :status    :error}
+    (< entered-value suggested-gas-price) {:hint-text (i18n/label :t/gas-price-lower-than
+                                                                  {:current
+                                                                   suggested-gas-price})
+                                           :status    :warning}
+    :else                                 {:hint-text (i18n/label :t/gas-price-current
+                                                                  {:current
+                                                                   current-gas-price})
+                                           :status    :default}))
+
+(defn view
+  []
+  (let [gas-price           (rf/sub [:wallet/tx-settings-gas-price])
+        current-gas-price   (rf/sub [:wallet/tx-settings-gas-price-route])
+        suggested-gas-price (rf/sub [:wallet/tx-settings-suggested-gas-price])
+        spectrum            {:low  (* 0.9 suggested-gas-price)
+                             :high (* 1.2 suggested-gas-price)}
```

Comment:
It would be great to have a const for these number to understand why we use these values (0.9, 1.2).

Reply:
We allow the user to go low (-10%) or high (+20%) of the suggested gas price. This logic is aligned with the Desktop. I will add a comment next to it. 👍

Reply:
We can still move it to constants right ? 

`(def low-10%)` and `(def high-20%)`